crossbeam-epoch/src/atomic.rs

```diff
@@ -936,6 +936,57 @@ impl<'g, T> Shared<'g, T> {
         &*self.as_raw()
     }
 
+    /// Dereferences the pointer.
+    ///
+    /// Returns a mutable reference to the pointee that is valid during the lifetime `'g`.
+    ///
+    /// # Safety
+    ///
+    /// * Dereferencing a pointer is unsafe because it could be pointing to invalid memory.
+    ///
```

Comment:
Maybe add another item saying "It is only valid to mutably dereference a pointer if no other threads will dereference it at the same time."

Reply:
Oh, I see you've added something similar below.